src-java/kilda-model/src/main/java/org/openkilda/model/bitops/cookie/CookieSchema.java

```diff
@@ -0,0 +1,107 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.model.bitops.cookie;
```

Comment:
Why did CookieSchema become a part of the bitops package? Shouldn't it be on the same level as the cookie entity? I'd propose to move the cookie and cookie schema under "org.openkilda.model.cookie".

Reply:
Move Cookie class and all related classes into `org.openkilda.model.cookie` was my initial idea. But it produces so huge change, that I have decided to do it in some late and as a PR that contains class movements only.

Reply:
Ok